gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/mapper/ApiLogsMapper.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.rest.api.management.v2.rest.mapper;
+
+import io.gravitee.rest.api.management.v2.rest.model.ApiLog;
+import io.gravitee.rest.api.model.v4.log.ConnectionLogModel;
+import java.time.OffsetDateTime;
+import java.util.List;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+import org.mapstruct.factory.Mappers;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Mapper(uses = { PlanMapper.class, ApplicationMapper.class })
+public interface ApiLogsMapper {
+    Logger logger = LoggerFactory.getLogger(ApiLogsMapper.class);
+    ApiLogsMapper INSTANCE = Mappers.getMapper(ApiLogsMapper.class);
+
+    @Mapping(source = "timestamp", target = "timestamp", qualifiedByName = "mapTimestamp")
+    ApiLog map(ConnectionLogModel connectionLog);
+
+    List<ApiLog> mapToList(List<ConnectionLogModel> logs);
+
+    @Named("mapTimestamp")
+    default OffsetDateTime mapTimestamp(String timestamp) {
+        return OffsetDateTime.parse(timestamp);
+    }
```

Comment:
We already have a "DateMapper" maybe we could add this method in it and use it?

Reply:
I didn't create a Date from the data coming from Elastic as I didn't need it to sort the result. However, to please OpenApi generated code I had to create an OffsetDateTime 🙈 
I can move the mapping method in the `DateMapper` (I didn't see it) or I can use a `DateTime` object in `ConnectionLogModel` 🤷